auth-table-based-service/pom.xml

```diff
@@ -96,7 +95,6 @@
               org.apache.cassandra.stargate,
               org.apache.cassandra.stargate.*,
               org.javatuples,
-              com.datastax.oss.driver.shaded.guava.*,
```

Comment:
For some reason this has to be removed; I think it would try to "pull-in" dependency from other modules instead of using a private copy. If we wanted to, we could alternatively Export shaded Guava from some other module, I suspect, but it took a while to get everything to work this way.

Reply:
OK I investigated here a bit more, I am kinda afraid this removal of the `com.datastax.oss.driver.shaded.guava.*,` and if it is good.. 

Bottom line is this module depends on the `core`, `authnz` and `persistance-api` modules.. And it imports set of different classes from these modules..
```
io.stargate.core.*,
io.stargate.auth,
io.stargate.db.*,
org.apache.cassandra.stargate,
org.apache.cassandra.stargate.*,
```

Now what happens if some of the modules above depend on the (1) shaded guava and (2) use shaded guava in some of the classes above.. I think then when this module tries to uses that classes, we would end up in the linkage errors.. Because the guava classes would be loaded with different class loader?

To overcome this, imo we should:
* remove the explicit dependency to the shaded guava (and thus depend on the version that `persistence-api` is providing)
* return the imported packages..

We need to do this for every module that depends on the `persistence-api`.. For other modules that don't, and thus don't have the shaded guava on their compile path we can declare explicitly..

Reply:
OK but here, we did not revert the import? and guava is in the `test` scope?

Reply:
Correct: Guava (shaded) is only needed by test code for this module. As to Import, as far as I understand it this is not needed for (unit) test code, only for runtime; this is why I did not add it back.

Reply:
I would still return the import.. Why? because tomorrow somebody can use a shaded guava in this module and he will forget to return the import! Plus, we would have the same approach in all the modules.. And this should be applied to all modules imo..